### DISCUSSION TITLE
Lack of establishment in bare ground simulations

### DISCUSSION THREAD

**Discussion by JessicaNeedham** [NGEET/fates#985] [General]
I’ve been running global FATES in fixed biogeography no comp mode and am finding that a bunch of  places that should have forests have extremely low biomass. I’m spinning up from bare ground and the problem seems to be lack of establishment. In some places starting the run in June rather than January will allow plants to grow. However, in most sites the lack of establishment seems to be caused by very high termination mortality in the first few time steps. I tried turning off all mortality except termination mortality, and then all but one termination mortality term at a time. It looks like in most places mortality is triggered by this line:
[https://github.com/NGEET/fates/blob/da1f7f7fd8ec1fc2ef70bb088d770a4004eabcf8/biogeochem/EDCohortDynamicsMod.F90#L816](https://github.com/NGEET/fates/blob/da1f7f7fd8ec1fc2ef70bb088d770a4004eabcf8/biogeochem/EDCohortDynamicsMod.F90#L816)

Some slides on my attempts to figure out this issue are here:
[https://docs.google.com/presentation/d/1tkhwVAYqVq_YYavTc65792AKrNRlqRBQsuOTk0mVjZ4/edit?usp=sharing](https://docs.google.com/presentation/d/1tkhwVAYqVq_YYavTc65792AKrNRlqRBQsuOTk0mVjZ4/edit?usp=sharing)

At the  FATES software meeting today @adrifoster reported seeing a similar lack of establishment in some NEON simulations. 

We discussed ways to make cold start simulations a bit more robust. Options included:
- supplemental seed rain in the first year (to get around the seasonality issues)
- increased storage in the first year
- no mortality in the first year

There is also the issue of what to do with plants that have negative storage. 

Noting this here so folks can chime in with suggestions.

**Comment by rosiealice** [comment]
So I guess the strange thing is that if the plants get past the first year, then they do relatively OK? This seems to suggest that either there is a problem around the initial conditions (of storage, temperature, water content, etc) or thst there is a problem just with the plants being small? 

I wonder thus if it's worth testing a larger minimum size, to see if simply growing larger (and maybe having more leaves and/or storage compared to other tissues) allow survival. 

- Also on that theme, it would be interesting to check an increase in the initial soil moisture. There seems to be a spike in hydraulic mortality at the beginning for lots of the sites which points to that being at least part of the issue sometimes?

**Reply by rgknox** [reply]
Could be that our allometry functions are not advantageous to small plants.  The default AGB relationships we typically use for tropical trees are really only relevant for larger trees, not saplings (ie Saldarriaga, Chave, etc).  If there is a large proportion of sapwood or fine-root compared with leaf, the CUE is going to be low. We've explored using different allometry functions for small and large, but it adds yet complexity.
If the nutrient model is active, one could specify a small starter leaf-to-fine-root scaling factor. This should give young plants a more favorable CUE. If they are not getting enough nutrient, they would just hopefully respond by growing slowly until they've optimized increased this scaling factor to an optimum level.

**Comment by rgknox** [comment]
I've seen this type of thing as well in tropical simulations.  Some thoughts:

- Could be that the initial trimming value is not conducive to success at these locations
- Could be water limitations associated with the initial soil water conditions, which were also observed by @mpaiao 
- Could be a phenology thing, plants that are instantiated during a period of the year with leaves off, or perhaps late in the leaf-out season are at a disadvantage

Some ideas that were "kicked" around on during the meeting:
- allow a "kick-start" period for cold-start simulations where we add some measures to prevent plant collapse
- kick-start period could be a few months to a few years
- we could relax the effects of the kick-start period, using full measures at the beginning, and let the effects decrease to have no-effect over the course of a year (to prevent shocks)
- the kick-start period could do things like: 
      - create a supplement to the plant carbon or nutrient gains to prevent storage from getting too low
      - simply turn off mortality

**Comment by adamhb** [comment]
@JessicaNeedham,

Just one other thought in addition to the great comments above. The difference between starting in January vs. June makes me think the issue could have something to do with the drought and cold phenology flags in the recruitment subroutine (see [here](https://github.com/NGEET/fates/blob/main/biogeochem/EDPhysiologyMod.F90#L2068))?

If January is a "cold" season then, depending on the pft, would start with leaves off which could lead to carbon starvation, which wouldn't happen in June? Just a thought!

**Reply by JessicaNeedham** [reply]
@adamhb , yes that's a great point and I think it is happening in at least some of the site. There could also be drought deciduousness PFTs starting with leaf off in the dry season. 

@rgknox is trialing a scheme where recruits are given additional carbon storage if needed for the first few years which will hopefully solve the problem of seasonality.

**Reply by adamhb** [reply]
I see, that sounds like it will be helpful.

**Comment by JessicaNeedham** [comment]
This issue has mostly been fixed by starting runs in June, increasing initial soil moisture and initializing the run with large trees (DBH 50 cm). 

I’ll make a  PR for the large tree size initialisation as it might be a useful feature for speeding up spin up in no comp (it won’t work in full fates). This works by setting the initial_density parameter as a negative and interpreting it to mean (positive) initial DBH, then calculating the cohort number density needed to give a closed canopy. 

I increased soil moisture in ELM here: https://github.com/E3SM-Project/E3SM/blob/b30a6a0cb3e610f80a00cd0fcdbfffab7044df54/components/elm/src/data_types/ColumnDataType.F90#L1619

by switching use_fates_planthydro to use_fates. Maybe we can make this the default? 

Results for a control, high soil moisture, large size, and large size + high soil moisture run are here: https://github.com/adrifoster/fates-global-cal/blob/main/jupyter_ppe_scripts/jessie_scripts/Atkin_dbhinit_soilmoisture.ipynb
Each panel is a grid cell where forest inventory data from Galbraith et al. 2013 or Piponoit et al. 2022 suggest there should be forest, but where default FATES (fixed biogeography no comp mode) had very low biomass. 

Even with all these changes some grid cells look like they will still have low biomass (I’ve only run for 10 years so far). There are also still a few grid cells that have reasonable GPP in SP mode but very low Veg C in fixed biogeography no comp mode, suggesting there is still something weird with the carbon balance in these sites.
